src/RIOPA.jl

```diff
@@ -1,7 +1,16 @@
-
 module RIOPA
 
-export parse_inputs
+export parse_inputs, read_config, generate_config
+
+import MPI
+
+const worldrank = Ref{Int}(-1)
```

Comment:
Question, do we want to use global at all?

Reply:
I was conflicted about this, too, and I'm certainly not tied to it. This is how a bunch of globals are defined in MPI.jl (like `COMM_WORLD`). I had a "conceptual" justification for it in that it is intuitively a global constant for each process. We could just as easily call a function everywhere we need to know the rank.